internal/vendors/service_test.go

```diff
@@ -52,3 +55,67 @@ func TestVendorService_GetSomeStuff(t *testing.T) {
 		})
 	}
 }
+
+func TestVendorService_GetAll(t *testing.T) {
+	sampleData := []Vendor{
+		{
+			Id:            1,
+			Name:          "name",
+			BpId:          1,
+			BpName:        "bp_name",
+			Rating:        1,
+			AreaGroupId:   1,
+			AreaGroupName: "group_name",
+			SapCode:       "sap_code",
+			ModifiedDate:  time.Now(),
+			ModifiedBy:    1,
+			Date:          time.Now(),
+		},
+	}
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	type fields struct {
+		mockVendorDBAccessor *MockvendorDBAccessor
+	}
+	type args struct {
+		ctx context.Context
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		want    []Vendor
+		wantErr bool
+	}{
+		// TODO: Add test cases.
+		{
+			name: "Test_GetAllVendor Success",
```

Comment:
tambah test false

Reply:
Fail test is just changing the "want" value and "wantErr" to true ya?  Not yet familiar with doing test ehehehe